### ISSUE TITLE
vtk-dicom already part of a normal VTK source build?

### ISSUE THREAD

**Issue by brmarkus** [dgobbi/vtk-dicom#231]
I just have built the latest VTK from source under Ubuntu (22.04), which shows the version "9.3.20240807".

I haven't enabled or changed any of the mentioned cmake config options from "https://github.com/dgobbi/vtk-dicom/blob/master/Documents/Installation.md"...

But I can see DICOM-parser CXX-files and the built libraries, also see DICOM tests in the test-suite.

I havent't checked the source codes of those "contained" DICOM-related files in VTK, I haven't compared the implementation to this repository...

Can you confirm that DICOM support has been added to VTK "natively" in the meantime (or the cmake build process or git-clone incorporate it in the meantime)?

In other words, is this repo `vtk-dicom` still needed when using VTK (built VTK from source)?
